src/test/java/com/mongodb/jdbc/MongoSQLResultSetMetaDataTest.java

```diff
@@ -0,0 +1,238 @@
+package com.mongodb.jdbc;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import org.bson.BsonType;
+import org.bson.BsonValue;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class MongoSQLResultSetMetaDataTest extends MongoSQLMock {
+    private static MongoSQLResultSetMetaData resultSetMetaData;
+
+    // __bot.a
+    private static int DOUBLE_COL = 1;
+    // __bot.str
+    private static int STRING_COL = 2;
+    // foo.a
+    private static int ANY_OF_INT_STRING_COL = 3;
+    // foo.b
+    private static int INT_NULLABLE_COL = 4;
+    // foo.c
+    private static int INT_COL = 5;
```

Comment:
Last time I reviewed this, I had to double check the `MongoSQLMock` to see why both of these return `ColumnNullable` for `isNullable`, since their names imply the former is nullable and the latter is not. This time, the same thing happened. Would you consider one or both of the following suggestions:
1. rename `INT_NULLABLE_COL` => `INT_OR_NULL_COL`
2. comment that `__bot.a`, `__bot.str`, `foo.c`, and `foo.d` _may_ be missing

I think either of those changes would help make this clear in just this file alone, without need to reference back to the mock class.